langserver/workspace_refs.go

```diff
@@ -189,10 +180,6 @@ func (h *LangHandler) externalRefsFromPkg(ctx context.Context, bctx *build.Conte
 	}()
 	span.SetTag("pkg", pkg)
 
-	pkgInWorkspace := func(path string) bool {
```

Comment:
If we now emit references to both internal and external definitions, you should change the name and docstring of this function.